pom.xml

```diff
@@ -33,8 +33,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>2.3.2</version>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.8</source>
+                    <target>1.8</target>
```

Comment:
I prefer to use the "maven.compiler.source" and "maven.compiler.target" properties and remove the plugin block.